launcher/internal/oci/interface.go

```diff
@@ -0,0 +1,45 @@
+// Package oci contains functionalities to interact with OCI image signatures.
+// https://github.com/opencontainers/image-spec/tree/main#readme.
+package oci
+
+// SigningAlgorithm is a specific type for string constants used for sigature signing and verification.
+type SigningAlgorithm string
+
+const (
+	// RSASSAPSS2048SHA256 is RSASSA-PSS 2048 bit key with a SHA256 digest supported for cosign sign.
+	RSASSAPSS2048SHA256 SigningAlgorithm = "RSASSA_PSS_2048_SHA256"
```

Comment:
No, our API enum does not specify bit length for a RSA public key. Since the current implementations use Tink to verify signatures and Tink only accepts at least 2048 bit key, I think it would be good to imply this length restriction from the naming. But to align with our API enum, we can keep the naming but change the value to `RSASSA_PSS_SHA256`. Done.

Reply:
I actually was wondering about the other direction: should we update/change our API enum? It's confusing to put the key size here but not in the API